src/sentry/rules/conditions/tagged_event.py

```diff
@@ -47,52 +51,66 @@ class TaggedEventCondition(EventCondition):
     def passes(self, event, state, **kwargs):
         key = self.get_option("key")
         match = self.get_option("match")
-        value = self.get_option("value")
+        value = self.get_option("value", "")
```

Comment:
Let's remove this change, and move `value = value.lower()` to after the check I proposed below.